src/components/UI/LanguageSelector/index.tsx

```diff
@@ -7,85 +16,94 @@ export enum FlagPosition {
 }
 
 interface Props {
-  className?: string
-  currentLanguage?: string
   flagPossition?: FlagPosition
-  handleLanguageChange: (language: string) => void
+  showWeblateLink?: boolean
+}
+
+const languageLabels: { [key: string]: string } = {
```

Comment:
moved these 2 constants outside the component so they are not initialized in every call to the function